src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -440,22 +383,20 @@ private static string WritePgWebBookmarks(IEnumerable<PostgresDatabaseResource>
                     sslmode = "disable"
                     """;
 
-            hash.Append(Encoding.UTF8.GetBytes(fileContent));
-
-            File.WriteAllText(Path.Combine(dir, $"{postgresDatabase.Name}.toml"), fileContent);
+            bookmarkFiles.Add(new ContainerFile
+            {
+                Name = $"{postgresDatabase.Name}.toml",
+                Contents = fileContent,
+                Mode = FileMode755,
```

Comment:
Only the folder would need 755, right? As it needs to be enumerated, but maybe I lack knowledge and the files too.

Reply:
Correct, I had these reversed.